test/tilejson.test.js

```diff
@@ -381,7 +381,7 @@ tape('findID', function(assert) {
     tape('should abort when the server takes too long', function(assert) {
         tilejson.getTile(1, 0, 0, function(err, data, headers) {
             assert.ok(err);
-            assert.equal(err.message, 'ETIMEDOUT');
+            assert.equal(err.message, 'ESOCKETTIMEDOUT');
```

Comment:
I think this change in error code is fine - it still timed out either way, which is what we want to see.